HelpfulItem.h

```diff
@@ -1,15 +1,15 @@
 #pragma once
 
 #include "Item.h"
-#include "Dwarf.h"
+//#include "Dwarf.h"
 #include "Character.h"
-#include "Paladin.h"
-#include "DragonSlayer.h"
-#include "Dragon.h"
+//#include "Paladin.h"
+//#include "DragonSlayer.h"
+//#include "Dragon.h"
```

Comment:
Remove dead code
You aren't using anything from `Character.h` in this header file.  Don't include what you don't use.  Forward declare instead
- [ ] revise